caesars-palace/blocks/carousel/carousel.js

```diff
@@ -385,6 +395,16 @@ export default async function decorate(block) {
       block.querySelectorAll('img').forEach((image) => {
         image.closest('picture').replaceWith(createOptimizedPicture(image.src, image.alt, false, [{ width: '960' }]));
       });
+
+      const firstSlide = document.getElementById('carousel-slide1');
```

Comment:
Seems the eager loading must be reset every time the media query matches (or it turns back to `lazy`). But here we know we need the two first slides loaded, so...

Reply:
Another good reason to move this logic into its own function since it does get called multiple times.